Formula/s/semgrep.rb

```diff
@@ -226,12 +226,9 @@ def install
     end
 
     ENV["SEMGREP_SKIP_BIN"] = "1"
-    python_path = "cli"
-    cd python_path do
-      venv = virtualenv_create(libexec, Formula["python@3.11"].bin/"python3.11")
-      venv.pip_install resources.reject { |r| r.name == "ocaml-tree-sitter" }
-      venv.pip_install_and_link buildpath/python_path
-    end
+    venv = virtualenv_create(libexec, "python3.11")
+    venv.pip_install resources
+    venv.pip_install_and_link buildpath/"cli"
```

Comment:
We could add an arg to `virtualenv_install_with_resources` to allow a path other than `buildpath`, but currently `semgrep` is the only formula that would use this feature.

Reply:
Could be nice. Probably worth waiting to see if we need it in additional formulae in the future.